opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/FilterQueryBuilderTest.java

```diff
@@ -937,93 +936,126 @@ void cast_to_string_in_filter() {
         dsl.equal(ref("string_value", STRING), dsl.castString(literal("1")))));
   }
 
+  private Float castToFloat(Object o) {
```

Comment:
nit: better to spell out the name of the argument: `Object object` or `Object obj`